crates/sui-graphql-rpc/src/functional_group.rs

```diff
@@ -94,14 +94,14 @@ fn functional_groups() -> &'static BTreeMap<(&'static str, &'static str), Functi
             (("Object", "balance"), G::Coins),
             (("Object", "balanceConnection"), G::Coins),
             (("Object", "coinConnection"), G::Coins),
-            (("Object", "defaultNameServiceName"), G::NameService),
+            (("Object", "defaultSuinsRegistration"), G::NameService),
             (("Object", "dynamicField"), G::DynamicFields),
             (("Object", "dynamicFieldConnection"), G::DynamicFields),
             // (("Object", "nameServiceConnection"), G::NameService),
```

Comment:
Looks like you would also need to uncomment this and rename it to `suinsRegistrations`?